config/locales/server.en.yml

```diff
@@ -92,6 +92,20 @@ en:
       not_allowed_theme: "`%{repo}` is not in the list of allowed themes (check `allowed_theme_repos` global setting)."
       ssh_key_gone: "You waited too long to install the theme and SSH key expired. Please try again."
       too_many_files: "The number of files (%{count}) in the theme has exceeded the maximum allowed number of files (%{limit})"
+      migrations:
+        invalid_filename: "Invalid filename for migration file: %{filename}. Filenames must begin with 4 digits followed by a hyphen and then a name that only contains alphanumeric characters with hyphens."
+        name_too_long: "Migration name is too long. It shouldn't exceed 150 characters."
```

Comment:
Are we able to avoid duplicating the `150` here and instead interpolate the constant? Otherwise, we risk this going out of sync when the limit is changed in the future.